bin/bento

```diff
@@ -69,10 +70,10 @@ class Options
         },
         argv: templates_argv_proc
       },
-      fix: {
```

Comment:
At the moment, the `bin/bento` code has only been in master about a week (and still not documented). I could add an alias here, but was hoping it was under (or not) used.